source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,157 @@
+
+
+    <div class="panel panel-drop panel-guide" id="accordion">
+    <div class="panel-heading panel-drop-heading">
+    <a class="accordion-toggle panel-drop-title collapsed" data-toggle="collapse" data-parent="#accordion" data-proofer-ignore data-target="#unique-anchor">
+      <h3 class="info panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-wrench"></span> CSR Details</h3>
+    </a>
+    </div>
+    <div id="unique-anchor" class="collapse" markdown="1" style="padding:10px;">
+    ### CSR Details Required {.info}
+
+    When requesting a CSR file, you must provide Pantheon Support with the following information:
+
+     - Common Name (e.g. `example.com`)
+     - DNS Names (e.g. `example.com`:`www.example.com`:`blog.example.com`:`dev.example.com`)
+     - Organization (e.g. `ACME, Inc.`)
+     - Organizational Unit \[optional\] (e.g. `Engineering`)
+     - Country (e.g. `US`)
+     - Locality (e.g. `San Francisco`)
+     - State / Province (e.g. `California`)
+     - Emails addresses \[optional\] (e.g. `someone@example.com`:`admin@example.com`)
+    </div>
+    </div>
+
+2. Pantheon support will provide you with the CSR file, to pass on to your **Certificate Authority** (CA).
+
+3. Once you have a set of certificates from the CA, send us:
+
+    - The end-client certificate
+    - Any intermediate certficates provided by the CA.<br><br>
+
+    Be sure to send these as separate files, not a "chained cert".
+
+4. [Test Before Going Live](#test-before-going-live) (optional, recommended)
+
+5. Disable Let's Encrypt by adding CAA DNS records TODO: fix link #disable-lets-encrypt-with-caa-records-required
+
+6. Update `A` and `AAAA` records provided by Pantheon support.
+
+
+## Test Before Going Live
+
+Test production domain(s) before updating DNS by overriding DNS on your local computer from your local `hosts` file:
+
+```bash
+23.185.0.X       example.com
+23.185.0.X       www.example.com
+```
+
+In the example `hosts` file above, replace `23.185.0.X` with the IP address provided by Pantheon and `example.com` with your domain name.
+
+
+For non-production domains, test on any environment (Dev, Test, Live or Multidev), just make sure to include the non-production domains on your certificate. We are happy to provide a new CSR if your original CSR and certificate did not initially non-production domains.
+
+## Disable Let's Encrypt with CAA Records (Required)
+
+A **Certification Authority Authorization** (CAA) record is used to specify which certificate authorities (CAs) are allowed to issue certificates for a domain. In order to ensure your custom certificate is served for all traffic, you must prevent Let’s Encrypt from issuing certificates. You have two options to prevent Let’s Encrypt from issuing certificates for domains on your custom certificate:
```

Comment:
You can also create a CAA policy that allows Let's Encrypt but whitelists the accounts that may issue. This would also prevent Pantheon from obtaining Let's Encrypt certs for the domain while still allowing the institution to use LE: https://community.letsencrypt.org/t/the-future-of-caa/62160